### PR TITLE
Display reviews on restaurant show

### PR DESCRIPTION
**Pull request by euniceYS**
RestaurantReviewForm and ReviewTile added to RestaurantShowContainer along with the Enzyme testing. Thanks!

### CODE REVIEW

File: app/assets/stylesheets/user-side.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.user-profile{
+  text-transform: uppercase;
+}
+
+.profile-page{
+  max-width: 1600px;
```
Comment (dpickett) [inline app/assets/stylesheets/user-side.css:6]: this is pretty high. What was the rationale in this setting?

File: app/javascript/react/components/RestaurantListTile.js [hunk 2]
```diff
@@ -20,14 +20,19 @@ class RestaurantListTile extends Component{
...
-          <h4>{this.props.phone_number}</h4>
+        <div className="rows restaurant_list_tile">
+          <div className="medium-4">
+            <img src={"https://s3-us-west-2.amazonaws.com/s.cdpn.io/331810/sample70.jpg"} alt="sample70" />
+          </div>
+          <div className="medium-8">
+            <h2>{this.props.name}</h2>
+            <ul>
+              <li className="rest-address">{this.props.address}</li>
+              {email}
```
Comment (dpickett) [inline app/javascript/react/components/RestaurantListTile.js:31]: these elements must appear inside the `<li>` for the HTML to bevalid

File: app/models/yelp_search.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+class YelpSearch
```
Comment (dpickett) [inline app/models/yelp_search.rb:1]: very nice! good use of a PORO here!

File: spec/javascript/containers/RestaurantShowContainerSpec.js [hunk 4]
```diff
@@ -32,45 +50,86 @@ describe('RestaurantShowContainer', () => {
...
+      });
       setTimeout(() => {
-        expect(wrapper.find('li.rest-phone-number').text()).toBe('603-867-5309');
-        done();
+        expect(wrapper.find(RestaurantTile).first().props().name).toEqual('Red Arrow Diner');
       }, 0);
     });
 
-    it('renders a div for restaurant photos', (done) => {
+    fit('renders a div for restaurant photos', (done) => {
```
Comment (dpickett) [inline spec/javascript/containers/RestaurantShowContainerSpec.js:130]: this test should be omitted or brought to passing state.

**Review by dpickett — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/user-side.css (hunk 1 above)
Comment (euniceYS) [reply app/assets/stylesheets/user-side.css:6]: I made this change on a new branch so we'll merge this branch and request another PR. Thanks!

File: app/javascript/react/components/RestaurantListTile.js (hunk 2 above)
Comment (euniceYS) [reply app/javascript/react/components/RestaurantListTile.js:31]: Updated the variables name on another branch that was mentioned above.

File: spec/javascript/containers/RestaurantShowContainerSpec.js (hunk 4 above)
Comment (euniceYS) [reply spec/javascript/containers/RestaurantShowContainerSpec.js:130]: updated on another branch.

**Pull request merged**